frameworks/helloworld/tests/test_sanity.py

```diff
@@ -142,28 +142,30 @@ def test_pod_list():
 @pytest.mark.sanity
 def test_pod_status_all():
     foldered_name = sdk_utils.get_foldered_name(config.SERVICE_NAME)
-    jsonobj = sdk_cmd.svc_cli(config.PACKAGE_NAME, foldered_name, 'pod status', json=True)
-    assert len(jsonobj) == config.configured_task_count(foldered_name)
-    for k, v in jsonobj.items():
-        assert re.match('(hello|world)-[0-9]+', k)
-        assert len(v) == 1
-        task = v[0]
-        assert len(task) == 3
-        assert re.match('(hello|world)-[0-9]+-server__[0-9a-f-]+', task['id'])
-        assert re.match('(hello|world)-[0-9]+-server', task['name'])
-        assert task['state'] == 'TASK_RUNNING'
+    jsonobj = sdk_cmd.svc_cli(config.PACKAGE_NAME, foldered_name, 'pod status --json', json=True)
+    assert jsonobj['service'] == foldered_name
```

Comment:
Looking at how we compare these outputs, could we rather specify a dict to which we want to compare?  That makes it a lot simpler to parse and extend?

Reply:
An issue is that other tests could add/remove pods, or the ordering may not be fixed. This is just checking that the resulting data looks valid rather than matching an exact value.